playbook/lib/generators/sample/sample_generator.rb

```diff
@@ -23,7 +23,7 @@ def update_yaml
       SAMPLES[category] = [@sample_name_underscore]
     end
 
-    File.open("app/pb_kits/playbook/data/samples.yml", "w") { |out| YAML.dump(samples_yaml, out) }
```

Comment:
When running the generator, I was hitting an error like this: 
```
/playbook/playbook/lib/generators/sample/sample_generator.rb:26:in `block in update_yaml': undefined local variable or method `samples_yaml' for #<SampleGenerator:0x00007ff7cd465538> (NameError)
```

This was my attempt to fix - the samples.yml file was updated with my new sample page after running

Reply:
Looks good!  Thanks for working this into your PR.